src/GameMarket.jsx

```diff
@@ -33,22 +33,25 @@ class GameMarket extends React.Component {
   handleTransact = (transaction) => {
     let shareCount = parseInt(this.state.shareCount);
     if (this.state.shareIndustryType !== 'None') {
-      const shareType = this.state.shareIndustryType;
+			let args = {
```

Comment:
Technically we could use `const` here since changing properties is not reassignment / rebinding, but `let` might be more readable since we are adding stuff to it later. Not sure what the best practice is.